static/app/utils/metrics/index.tsx

```diff
@@ -175,10 +177,22 @@ export type MetricMetaCodeLocation = {
 
 export type MetricCorrelation = {
   duration: number;
+  metricSummaries: {
+    count: number;
+    max: number;
+    min: number;
+    spanId: string;
+    sum: number;
+  }[];
```

Comment:
Are they all required? For `_metrics_summary` all the properties can be `null`. I would assume the same could theoretically happen here for `count`, `max`, `min` and `sum`.